packages/server/utils/getS3PutUrl.ts

```diff
@@ -0,0 +1,8 @@
+import protocolRelativeUrl from './protocolRelativeUrl'
+import path from 'path'
+import {APP_CDN_USER_ASSET_SUBDIR} from 'parabol-client/utils/constants'
+
+export default function getS3PutUrl(partialPath) {
+  const baseUrl = protocolRelativeUrl.parse(process.env.CDN_BASE_URL)
+  return path.join(baseUrl.pathname, APP_CDN_USER_ASSET_SUBDIR, partialPath)
+}
```

Comment:
will be deprecated

Reply:
feel free to put deprecated notes as comments into the code itself! that way we're more likely to remove it in the future